Makefile

```diff
@@ -294,3 +304,7 @@ bluetoothdevice: clean
 .PHONY: bluetoothdevice_image
 bluetoothdevice_image:bluetoothdevice
 	docker build -t bluetooth_mapper:v1.0 ./mappers/bluetooth_mapper/
+
+# no-phony target
+$(MISSPELL):
+	$(call fetch_tool,github.com/client9/misspell/cmd/misspell,$(MISSPELL_VERSION))
```

Comment:
Misspell is already included in golangci-lint, can we simply use golangci-lint with `-E misspell`?

Reply:
Yeah we can, but need to add config file to ignore specific word "mosquitto" since linter config can not be specified by cmd-line.